bigdata-core/bigdata-sails/src/java/com/bigdata/rdf/sail/BigdataSail.java

```diff
@@ -4854,12 +4853,11 @@ public void begin() throws SailException {
          * <p>
          * {@inheritDoc}
          */
-//FIXME:  Sesame 2.8 not used for 2.1.4 Release
-//		@Override
-//		public void begin(IsolationLevel level)
-//				throws UnknownSailTransactionStateException, SailException {
-//
-//		}
+		@Override
+		public void begin(IsolationLevel level)
+				throws UnknownSailTransactionStateException, SailException {
+
```

Comment:
Why not call begin to be consistent? Or more likely, call begin(level) with the configured default level from begin().

Reply:
AFAIK BigdataSailConnection is extended only by two private classes in this very file, so both begins seem to be no-ops anyway. But I can convert one into calling another, for consistency.

Reply:
Docs here: https://bitbucket.org/openrdf/sesame/src/71eb79608cd97696598fce5cf85bfd8aad641e74/core/sail/api/src/main/java/org/openrdf/sail/SailConnection.java?at=sesame-2.8.11&fileviewer=file-view-default#SailConnection.java-180 say that UnknownSailTransactionStateException should be thrown if unsupported level is supplied, the problem is they have different enum than Blazegraph does, so I am not sure how to make sense of all that.